tests/FSharp.Compiler.ComponentTests/FSharpChecker/TransparentCompiler.fs

```diff
@@ -894,4 +895,49 @@ let ``LoadClosure for script is recomputed after changes`` () =
         |> Seq.map (fun (k, g) -> k, g |> Seq.map fst |> Seq.toList)
         |> Map
 
-    Assert.Equal<JobEvent list>([Weakened; Requested; Started; Finished; Weakened; Requested; Started; Finished], closureComputations["FileFirst.fs"])
\ No newline at end of file
+    Assert.Equal<JobEvent list>([Weakened; Requested; Started; Finished; Weakened; Requested; Started; Finished], closureComputations["FileFirst.fs"])
+    
+[<Fact>]
+let ``TryGetRecentCheckResultsForFile returns None before first call to ParseAndCheckFileInProject`` () =
+    let project = SyntheticProject.Create(
+        sourceFile "First" [])
+    
+    ProjectWorkflowBuilder(project, autoStart = false, useTransparentCompiler = true) {
```

Comment:
With `autoStart = false` the workflow won't be automatically executed and will just return `Async<...>` which you need to start yourself (and now it's just ignored). And when you do it will still to the initial save & check. 

Probably what you want here is either `clearCache` or `updateFile "First"` before `tryGetRecentCheckResults`.

Reply:
Thanks, that was exactly the case.